src/wdio/services/TerraCommands/determine-test-options.js

```diff
@@ -49,11 +49,12 @@ const axeOptions = (args) => {
     options, viewports,
   } = determineArgs(args);
 
+  const globalAxeRules = global.browser.options.axe.options.rules;
   const axeRules = options.rules || options.axeRules;
 
   return {
     ...viewports && { viewports },
-    ...axeRules && { rules: axeRules },
+    ...axeRules && { rules: { ...globalAxeRules, ...axeRules } },
```

Comment:
I'm not sure this change is needed. Did the axe tool not properly configure with the root level configuration? Do you have a couple test runs as evidence you can link?